Mimir/Repositories/AllRuneRepository.cs

```diff
@@ -0,0 +1,91 @@
+using Lib9c.GraphQL.Enums;
+using Libplanet.Crypto;
+using Mimir.Models.Assets;
+using Mimir.Services;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace Mimir.Repositories;
+
+public class AllRuneRepository
+{
+    private readonly MongoDBCollectionService _mongoDBCollectionService;
+    private readonly Dictionary<PlanetName, IMongoCollection<BsonDocument>> _collections = new();
+    private readonly Dictionary<PlanetName, IMongoDatabase> _databases = new();
+
+    public AllRuneRepository(MongoDBCollectionService mongoDbCollectionService)
+    {
+        _mongoDBCollectionService = mongoDbCollectionService;
+
+        const string collectionName = "all_rune";
+        foreach (var planetName in new[] { PlanetName.Heimdall, PlanetName.Odin })
+        {
+            var databaseName = PlanetNameToDatabaseName(planetName);
+            var collection = _mongoDBCollectionService.GetCollection<BsonDocument>(
+                collectionName,
+                databaseName);
+            _collections[planetName] = collection;
+
+            var database = _mongoDBCollectionService.GetDatabase(databaseName);
+            _databases[planetName] = database;
+        }
+    }
+
+    public List<Rune>? GetRunes(PlanetName planetName, Address avatarAddress)
+    {
+        var collection = GetCollection(planetName);
+        var filter = Builders<BsonDocument>.Filter.Eq("Address", avatarAddress.ToHex());
+        var document = collection.Find(filter).FirstOrDefault();
+        if (document is null)
+        {
+            return null;
+        }
+
+        try
+        {
+            var runesDoc = document["State"]["Object"]["Runes"].AsBsonDocument;
+            return runesDoc
+                .Select(rune => rune.Value.AsBsonDocument)
+                .Select(runeValue => new Rune(runeValue))
+                .ToList();
+        }
+        catch (KeyNotFoundException)
+        {
+            return null;
+        }
+    }
+
+    private static string PlanetNameToDatabaseName(PlanetName planetName)
```

Comment:
Since we're still migrating, let's create `DiffBaseRepositories` by duplicating `BaseRepositories` to allow other classes to use it.

Reply:
I've created a `DiffRepository` in another branch that I worked on based on this branch.(https://github.com/planetarium/mimir/pull/125/files#diff-8009cbb12596d2a5e471469d2163967630f479bb3308e49c766b5bdd188c728b)
I think we can continue to improve it from there.🙂